webapp-django/crashstats/settings/base.py

```diff
@@ -125,13 +127,20 @@ def path(*dirs):
     'crashstats.tokens.middleware.APIAuthenticationMiddleware',
     'crashstats.crashstats.middleware.Pretty400Errors',
 )
+MIDDLEWARE = list(MIDDLEWARE)
+
+if os.environ.get('REMOVED_MIDDLEWARE') is not None:
+    REMOVED_MIDDLEWARE = config('REMOVED_MIDDLEWARE', cast=Csv())
+    for middleware_to_remove in REMOVED_MIDDLEWARE:
+        MIDDLEWARE.remove(middleware_to_remove)
```

Comment:
I don't think we want to have an environment variable that can remove certain middleware; I could see a bad config value removing a middleware that's critical to the site functioning, and we only need to remove one middleware, and only during tests. We should be a bit more conservative in what we allow to be changed.

`webapp-django/crashstats/settings/test.py` contains some extra settings that get applied on top of `base.py` during test runs (this works because we tell Django that the settings are at `crashstats.settings`, which imports `webapp-django/crashstats/settings/__init__.py`, where the magic is). In `test.py` we could modify `MIDDLEWARE` to remove `SessionRefresh` manually. That way there's no config at all that can modify the middleware list.